### PR TITLE
how-to/maintain: change br best practices to use cases

### PR DESCRIPTION
**Pull request by TomShawn**
### What is changed, added or deleted? 

+ Change BR Best Practices to BR Case Demonstration.
+ Refine the language and layout.

### What is the related PR or file link(s)? 

- [x] Reference link(s): https://github.com/pingcap/docs-cn/pull/2242, [Original BR best practices](https://pingcap.com/docs-cn/dev/reference/tools/br/br-best-practices/)
- [ ] This PR is to align with:
- [ ] N/A (not applicable)

### Which TiDB version(s) does your changes apply to? 

- [ ] All active versions: dev, v3.0, v2.1, v3.1
- [x] dev (the latest development version)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)
- [x] v3.1 (TiDB 3.1 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] All active and inactive versions
- [ ] N/A (not applicable)

**Note:** If your changes apply to multiple TiDB versions, make sure you update the documents in the **corresponding** version folders such as "dev", "v3.0", "v2.1" and "v3.1" in this PR.

- [ ] Updated one version first. Will update other versions after I get two LGTMs.

### CODE REVIEW

**Comment by TomShawn** [comment]
@3pointer @overvenus @GregoryIan PTAL, thanks!

File: dev/reference/tools/br/br.md [hunk 1]
```diff
@@ -310,7 +310,7 @@ br restore table \
 - 在使用共享存储时，推荐使用高吞吐的存储硬件，因为存储的吞吐会限制备份或恢复的速度。
 - 推荐在业务低峰时执行备份操作，这样能最大程度地减少对业务的影响。
 
-更多最佳实践内容，参见 [BR 最佳实践文档](/dev/reference/tools/br/br-best-practices.md)。
+更多最佳实践内容，参见 [BR 备份恢复案例展示](/dev/reference/tools/br/br-case-demo.md)。
```
Comment (3pointer) [inline dev/reference/tools/br/br.md]:
```suggestion
更多案例展示内容，参见 [BR 备份恢复案例展示](/dev/reference/tools/br/br-case-demo.md)。
```

Comment (TomShawn) [reply dev/reference/tools/br/br.md]: 这里实际上就是“最佳实践”的内容，只不过放进了“案例展示”的文档中。

**Comment by TomShawn** [comment]
@kissmydb PTAL, thanks!

File: dev/TOC.md [hunk 2]
```diff
@@ -62,7 +62,7 @@
     + 备份与恢复
       - [使用 Mydumper/Loader 进行备份与恢复](/dev/how-to/maintain/backup-and-restore/mydumper-loader.md)
       - [使用 BR 进行备份与恢复](/dev/reference/tools/br/br.md)
-      - [BR 备份与恢复最佳实践](/dev/reference/tools/br/br-best-practices.md)
+      - [BR 备份恢复案例展示](/dev/reference/tools/br/br-case-demo.md)
```
Comment (lilin90) [inline dev/TOC.md]: The word `案例` is inappropriate to me. I think those use cases are more like `使用场景` in Chinese, not sharing content from our users.
```suggestion
      - [BR 备份与恢复场景示例](/dev/reference/tools/br/use-cases.md)
```

File: dev/reference/tools/br/br-case-demo.md [hunk 3]
```diff
@@ -1,80 +1,75 @@
 ---
-title: BR 快速备份恢复最佳实践
-category: how-to
+title: BR 备份恢复案例展示
```
Comment (lilin90) [inline dev/reference/tools/br/br-case-demo.md]: Please update the title and file name for all occurrences in this PR. Thanks!

File: dev/reference/tools/br/br-case-demo.md [hunk 4]
```diff
@@ -1,80 +1,75 @@
...
+category: reference
+aliases: ['/docs-cn/dev/how-to/maintain/backup-and-restore/br-best-practices/']
 ---
 
-# BR 快速备份恢复最佳实践
+# BR 备份恢复案例展示
 
-[Backup & Restore](/dev/reference/tools/br/br.md)（下文简称 BR）是 PingCAP 新推出的分布式快速备份和恢复工具。本文描述了 BR 在备份和恢复场景下的操作过程以及注意事项，供用户参考以达到最佳实践。
+[Backup & Restore](/dev/reference/tools/br/br.md)（下文简称 BR）一款分布式的快速备份和恢复工具。本文展示了四种备份和恢复场景下的 BR 操作过程，以帮助读者达到以下目标：
```
Comment (lilin90) [inline dev/reference/tools/br/br-case-demo.md]: - I would want to know what are the four use cases at the beginning. That would help users to know the whole structure and to choose which part they would like to read.
- Add a link at least, or directly show the four use cases at the beginning.

```suggestion
[Backup & Restore](/dev/reference/tools/br/br.md)（下文简称 BR）一款分布式的快速备份和恢复工具。本文展示了[四种备份和恢复场景](#使用场景)下的 BR 操作过程，以帮助读者达到以下目标：
```

File: dev/reference/tools/br/br-case-demo.md [hunk 5]
```diff
@@ -197,16 +193,14 @@ bin/br backup table --db batchmark --table order_line -s local:///br_data/ --pd
...
-| 数据大小        | `total size(MB): 353227.18`             |
-| 备份吞吐        | `avg speed(MB/s)` 从 `358.09` 提升到 `659.59`    |
-| 单 TiKV 实例吞吐 | `avg speed(MB/s)/tikv_count` 从 `89` 提升到 `164.89` |
+* 备份耗时：`total take(s)` 从 `986.43` 减少到 `535.53`
+* 数据大小：`total size(MB): 353227.18`
+* 备份吞吐：`avg speed(MB/s)` 从 `358.09` 提升到 `659.59`
+* 单个 TiKV 实例的吞吐：`avg speed(MB/s)/tikv_count` 从 `89` 提升到 `164.89`
 
-### 从网络盘恢复备份数据
+### 恢复网络盘中的备份数据
```
Comment (lilin90) [inline dev/reference/tools/br/br-case-demo.md]: Keep the heading with that in `使用场景` consistent. And also update other places in this PR.

```suggestion
### 恢复网络盘中的备份数据（推荐）
```

File: dev/reference/tools/br/br-case-demo.md [hunk 6]
```diff
@@ -1,80 +1,75 @@
...
 ## 使用场景
 
 本文描述以下四种使用场景：
 
 * 将单表数据备份到网络盘（推荐）
-* 从网络盘的备份进行恢复（推荐）
+* 恢复网络盘中的备份数据（推荐）
 * 将单表数据备份到本地磁盘
-* 从本地磁盘的备份进行恢复
+* 恢复本地磁盘中的备份数据
```
Comment (lilin90) [inline dev/reference/tools/br/br-case-demo.md]: Please add links to this four use cases respectively.

**Comment by TomShawn** [comment]
@lilin90 Comments addressed, PTAL again, thanks!

File: dev/reference/tools/br/use-cases.md [hunk 7]
```diff
@@ -1,80 +1,75 @@
 ---
-title: BR 快速备份恢复最佳实践
-category: how-to
+title: BR 备份与恢复场景示例
+category: reference
+aliases: ['/docs-cn/dev/how-to/maintain/backup-and-restore/br-best-practices/']
```
Comment (lilin90) [inline dev/reference/tools/br/use-cases.md]: @TomShawn I find it lacks one alias for `/dev/reference/tools/br/br-best-practices.md`. This lead to a 404 for the original link: https://pingcap.com/docs-cn/dev/reference/tools/br/br-best-practices/

Please help update it. Thanks!

**Pull request merged**